package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "channelape-sdk",
-  "version": "1.48.0",
```

Comment:
we try to append .develop-x where x is the iteration we are attempting to publish(this part should be handled by CI as long as the version ends with .develop-0). NPM doesn't support snapshots, so once you publish a version you can't remove. Not super important, since we are targeting specific versions in our services but just how we get around publishing production versions.

Reply:
Should I leave it as 1.49.0 then?

Reply:
all i was getting at was if you make it 1.49.0.develop-0 Ci will publish with that version and continue to increment develop as long as it's the same major minor. It's how we test like the webapp and playbook with it in staging.